src/java/org/apache/cassandra/streaming/StreamingState.java

```diff
@@ -34,7 +34,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.cassandra.cache.IMeasurableMemory;
 import org.apache.cassandra.db.virtual.SimpleDataSet;
+import org.apache.cassandra.locator.InetAddressAndPort;
```

Comment:
your build is failing https://app.circleci.com/pipelines/github/NateAdere/cassandra/77/workflows/1c5a9c50-d8b0-4f34-91a2-f591816b86d9/jobs/1733